docker/go/Dockerfile

```diff
@@ -6,7 +6,7 @@
 FROM ubuntu:bionic AS builder
 
 ARG PULUMI_VERSION
-ARG GO_RUNTIME_VERSION=1.21.1
+ARG LANGUAGE_VERSION
 ENV GO_RUNTIME_386_SHA256 b93850666cdadbd696a986cf7b03111fe99db8c34a9aaa113d7c96d0081e1901
 ENV GO_RUNTIME_AMD64_SHA256 b3075ae1ce5dab85f89bc7905d1632de23ca196bd8336afd93fa97434cfa55ae
 ENV GO_RUNTIME_ARM64_SHA256 7da1a3936a928fd0b2602ed4f3ef535b8cd1990f1503b8d3e1acc0fa0759c967
```

Comment:
I'm guessing you're already well aware, but in case you're not, we'll have to deal with these SHAs as part of the subsequent change to matrix the Go images.